packages/forms/frontend/sirius-components-forms/src/views/DiagramFilterView.tsx

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { WorkbenchViewComponentProps } from '@eclipse-sirius/sirius-components-core';
+import { makeStyles } from '@material-ui/core';
+import { GQLForm, GQLWidgetSubscription } from '../form/FormEventFragments.types';
+import { Group } from '../groups/Group';
+import { useDetailsViewConfiguration } from './DetailsViewConfiguration';
+import { DiagramFilterViewProps } from './DiagramFilterView.types';
+import { FormBasedView } from './FormBasedView';
+
+const useDiagramFilterViewStyles = makeStyles((theme) => ({
+  content: {
+    padding: theme.spacing(1),
+  },
+}));
+
+export const DiagramFilterView = (props: DiagramFilterViewProps) => {
```

Comment:
To be moved in the frontend module `sirius-web-application` along with its `.types.ts` file. Don't call it a view, it's not a workbench view like the explorer or the details view. Use a name like `DiagramPanelFilterAction` instead.

Reply:
Done, renamed it to "DiagramFilter" and the component containing the form to "DiagramFilterForm"